cmd/kubeless/autoscaleCreate.go

```diff
@@ -0,0 +1,62 @@
+package main
+
+import (
+	"github.com/Sirupsen/logrus"
+	"github.com/kubeless/kubeless/pkg/utils"
+	"github.com/spf13/cobra"
+)
+
+var autoscaleCreateCmd = &cobra.Command{
+	Use:   "create <name> FLAG",
+	Short: "automatically scale function based on monitored metrics",
+	Long:  `automatically scale function based on monitored metrics`,
+	Run: func(cmd *cobra.Command, args []string) {
+		if len(args) != 1 {
+			logrus.Fatal("Need exactly one argument - function name")
+		}
+		funcName := args[0]
+
+		min, err := cmd.Flags().GetInt32("min")
+		if err != nil {
+			logrus.Fatal(err)
+		} else if min == 0 {
+			logrus.Fatalf("--min can't be 0")
+		}
+		max, err := cmd.Flags().GetInt32("max")
+		if err != nil {
+			logrus.Fatal(err)
+		} else if min == 0 {
```

Comment:
1. this is comparing against `min`, not `max`
2. should be `else if max < min`

Reply:
👍